packages/cli/src/commands/auth/login.spec.ts

```diff
@@ -75,12 +75,12 @@ describe('auth:login', () => {
   test
     .command(['auth:login', '-e', 'foo'])
     .catch(/Expected --environment=foo/)
-    .it('reject invalid environment');
+    .it('reject invalid environment', async () => {});
```

Comment:
I noted timing issue and promise timing out here.
I'm honestly not too sure of what's going in in this tests, the syntax is really painful to read.